src/providers/toast.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import { ToastController } from 'ionic-angular';
+
+@Injectable()
+export class ToastService {
+    private faveToast; 
+    constructor(private toast: ToastController) { } 
+    
+    favoriteToast(routeOrStop: string, isFave: boolean): void {
+        if (isFave){ 
```

Comment:
We can cut out some code here.  How about something like
```typescript
  let txt = `${routeOrStop} ${isFave ? 'added' : 'removed'} from favorites`;
  this.toast.create(message: txt, ...);
  ...
```
The langage allows us to interpolate into strings like so `${someCode or aVariable}`, but only when said string is initialized using backticks (`)